src/applications/claims-status/tests/e2e/08.claim-letters-keyboard-only.cypress.spec.js

```diff
@@ -27,14 +27,14 @@ describe('Claim Letters Page', () => {
     cy.contains(/Previous/i).should('not.exist');
 
     // Click 'Next' button
-    cy.tabToElement('.pagination-next li .button-next').realPress('Enter');
+    cy.tabToElement('.usa-pagination__next-page').realPress('Enter');
 
     // Now on second page
     // 'Previous' button should now be shown
     cy.contains(/Previous/i).should('exist');
 
     // Click 'Prev' button
-    cy.tabToElement('.pagination-prev li .button-prev', false, true) // had to use forward = false parameter to avoid a timeout error
+    cy.tabToElement('.usa-pagination__previous-page', false, true) // had to use forward = false parameter to avoid a timeout error
```

Comment:
Fixing some e2e tests that are broken now that v3 web-components are the default